src/graphql/queries/events.ts

```diff
@@ -1,24 +1,63 @@
 import gql from 'graphql-tag';
 
 export interface EventType {
+  id: string;
   name: string;
   eventDate: Date;
+  // ambulances: Ambulance[];
+  // hospitals: Hospital[];
 }
 
 export const FETCH_ALL_EVENTS = gql`
```

Comment:
both this query and `GET_ALL_EVENTS` are already used on other pages - can you check that these changes don't affect them in any way

Reply:
I commented them out because `Ambulance` and `Hospital` were undefined and I thought that would be fixed once `resource-management` was merged into master. But i'll just go ahead and add them.